lib/config/options/index.ts

```diff
@@ -364,6 +364,13 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     type: 'string',
   },
+  {
+    name: 'useSqliteCache',
+    description:
+      'If set to `true`, Renovate uses a SQLite database for its cache instead of the default file system.',
+    globalOnly: true,
+    type: 'boolean',
```

Comment:
```suggestion
    type: 'boolean',
    experimental: true,
```

let's make it experimental, so we unify to single cache type option later